### PR TITLE
Cat and cluwne bans

### PR DESCRIPTION
**Pull request by tortellinitony**
Will now offer bodies to ghosts when converted to antag.

### CODE REVIEW

File: code/controllers/subsystem/jobs.dm [hunk 1]
```diff
@@ -255,6 +255,10 @@ var/datum/controller/subsystem/job/SSjob
 
 	HandleFeedbackGathering()
 
+	for(var/mob/dead/new_player/player in unassigned)
```
Comment (ghost) [inline code/controllers/subsystem/jobs.dm]: Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)

File: code/controllers/subsystem/jobs.dm [hunk 2]
```diff
@@ -255,6 +255,10 @@ var/datum/controller/subsystem/job/SSjob
...
+		if(jobban_isbanned(player, "catban") || jobban_isbanned(player, "cluwneban"))
```
Comment (ghost) [inline code/controllers/subsystem/jobs.dm]: Could we turn these `"catban"` and `"cluwneban"` into defines? much like `if(jobban_isbanned(owner, ROLE_SERVANT_OF_RATVAR)`

**Review by ghost — CHANGES_REQUESTED** [review]
We also need to make sure cat-banned players can not use ghost roles and ghost role spawners.

**Comment by j-mie** [comment]
build this

**Comment by ghost** [comment]
build this

**Review by j-mie — APPROVED** [review]

File: hippiestation.dme [hunk 3]
```diff
@@ -2061,8 +2062,6 @@
 #include "code\orphaned_procs\dbcore.dm"
 #include "code\orphaned_procs\priority_announce.dm"
 #include "code\orphaned_procs\statistics.dm"
-#include "hippiestation\code\_DEFINES\DNA.dm"
```
Comment (j-mie) [inline hippiestation.dme:2064]: ?

Comment (tortellinitony) [reply hippiestation.dme:2064]: moved all defines to the actual defines folder

**Comment by tortellinitony** [comment]
I'll fix the conflicts later. I can add a separate check from the jobbans so that it doesn't conflict maybe?

**Comment by ghost** [comment]
Remember to put your changelog in the PR description

**Comment by tortellinitony** [comment]
This PR will need to be remade since some asshole changed jobs.dm to job.dm for no reason

**Pull request closed without merging**